src/runner/centralized_kb_hook.cpp

```diff
@@ -22,6 +23,30 @@ namespace CentralizedKeyboardHook
     std::mutex mutex;
     HHOOK hHook{};
 
+    // To store information about handling pressed keys.
+    struct PressedKeyDescriptor
+    {
+        DWORD virtualKey; // Virtual Key code of the key we're keeping track of.
+        std::wstring moduleName;
+        std::function<bool()> action;
+        UINT_PTR idTimer; // Timer ID for calling SET_TIMER with.
+        UINT millisecondsToPress; // How much time the key must be pressed.
+        bool operator<(const PressedKeyDescriptor& other) const
+        {
+            // We'll use the virtual key as the real key, since looking for a hit with the key is done in the more time sensitive path (low level keyboard hook).
+            return virtualKey < other.virtualKey;
+        };
+    };
+    std::multiset<PressedKeyDescriptor> pressedKeyDescriptors;
```

Comment:
If the order is not important, perhaps we could be more explicit about it and use `std::unordered_multiset` instead? It should also boost performance a bit.

Reply:
`unordered_multiset` would make inserts faster and lookups for vk_code slower, I think.
I think we want faster lookups, which will happen more times and during the critical keyboard hook path.
Inserts happen less frequently.

Reply:
perhaps we could use unordered_multimap then? just a thought 🙂

Reply:
That could be better :) Though I don't think it's worth the effort here (at least for now) and this way we use the same structure used in the other map.